isle-sofa-boot-starter/src/main/java/com/alipay/sofa/isle/spring/listener/SofaModuleBeanFactoryPostProcessor.java

```diff
@@ -55,7 +55,7 @@ public void postProcessBeanFactory(ConfigurableListableBeanFactory beanFactory)
                                                                   Class... types) {
         Map<String, BeanDefinition> map = new HashMap<>();
         for (Class type : types) {
-            String[] beanNamesForType = beanFactory.getBeanNamesForType(type);
+            String[] beanNamesForType = beanFactory.getBeanNamesForType(type, true, false);
```

Comment:
It seems a bug here, we should transform the returned beanNames via `BeanFactoryUtils.transformedBeanName`, otherwise, it would ignore the beans which implement `FactoryBean` or defined in Configuration class.

Reply:
fixed